CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 
 ## Unreleased
 
+## [26.4.6] - 2021-06-22
+
+### Added
+
+- SC-9148 - Add migration for change of school year on BRB
```

Comment:
Where is a filter to only execute this in BRB instance?
What blocks us to have this somewhere like in the superhero?